rslib/src/links.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright: Ankitects Pty Ltd and contributors
+// License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+
+use strum::{Display, EnumIter, EnumString};
+
+static HELP_SITE: &'static str = "https://docs.ankiweb.net/";
+
+#[derive(Debug, PartialEq, Clone, Copy, Display, EnumIter, EnumString)]
+pub enum HelpPage {
+    #[strum(serialize = "getting-started#note-types")]
```

Comment:
I'd probably go with a separate impl like to_page() on the enum that returns the relevant help page, instead of relying on serialize here. It just feels a bit odd to be returning webpage fragments in a derive, and a match expr would be a bit easier to read, eg BrowsingFindAndReplace => "browsing#find-and-replace".